### PR TITLE
[#3223] Volunteer hours  spent in last 30 days

### LINKED ISSUE #3223: Show hours volunteer has spent in the last 30 days
**What type(s) of user does this feature affect?**
 - volunteers
 - supervisors
 - admins

**Description**
Show hours volunteer has spent in the last 30 days on the index view

**Screenshots of current behavior, if any**  
You can paste images on the clipboard here
Screen Shot 2022-02-18 at 11 49 53 AM

_QA Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin

password for all users: 12345678  

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by ptrela**
### What github issue is this PR for, if any?
Resolves #3223

### What changed, and why?
Added "Hours spent in last 30 days" column inside volunteer index page.

### How will this affect user permissions?
- Volunteer permissions:
it won't
- Supervisor permissions:
it won't
- Admin permissions:
it won't

### How is this tested? (please write tests!) 💖💪
Tests will be written before opening pull request

### Screenshots please :)
Soon :)

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

File: app/datatables/volunteer_datatable.rb [hunk 1]
```diff
@@ -26,7 +27,8 @@ def data
           case_id: volunteer.most_recent_attempt_case_id,
           occurred_at: I18n.l(volunteer.most_recent_attempt_occurred_at, format: :full, default: nil)
         },
-        supervisor: {id: volunteer.supervisor_id, name: volunteer.supervisor_name}
+        supervisor: {id: volunteer.supervisor_id, name: volunteer.supervisor_name},
+        hours_spent_in_days: volunteer.hours_spent_in_days(30)
```
Comment (compwron) [inline app/datatables/volunteer_datatable.rb:31]: Why is this hardcoded to 30?

**Comment by compwron** [comment]
How's it going? I am excited to merge this :)

File: app/datatables/volunteer_datatable.rb (hunk 1 above)
Comment (compwron) [reply app/datatables/volunteer_datatable.rb:31]: Ah I see, we are assuming that there are 30 days in a month

**Review by compwron — APPROVED** [review]
Let's do it

**Comment by ptrela** [comment]
@compwron Oh, sorry, i was offline last days :)
I wanted to write some specs for this task, maybe i can do it in some seperate pull request without issue? :)

Is everything okay with this task? :)

**Pull request merged**